iis-common/src/main/java/eu/dnetlib/iis/common/pig/udfs/DeduplicateIdsWithConfidence.java

```diff
@@ -13,55 +13,76 @@
 import org.apache.pig.impl.logicalLayer.schema.Schema;
 
 /**
- * Deduplicates bag of tuples where tuple[0] is identifier and tuple[1] is confidence level.
- * Highest confidence level is picked when identifier duplicate is found.
- * Identifiers are sorted lexicographically.
+ * Deduplicates bag of tuples where tuple[0] is identifier and tuple[confidenceLevelPosition] is confidence level. 
+ * Highest confidence level is picked when identifier duplicate is found. Identifiers are sorted lexicographically.
  *
  * @author mhorst
  */
 public class DeduplicateIdsWithConfidence extends EvalFunc<DataBag> {
 
+    /**
```

Comment:
Well, what about simply IdConfidenceTupleDeduplicator? You have written in comments how it is actually deduplicated.